go/vt/vttablet/customrule/filecustomrule/filecustomrule.go

```diff
@@ -34,12 +34,22 @@ import (
 var (
 	// Actual FileCustomRule object in charge of rule updates
 	fileCustomRule = NewFileCustomRule()
-	// Commandline flag to specify rule path
-	fileRulePath = flag.String("filecustomrules", "", "file based custom rule path")
 
-	fileRuleShouldWatch = flag.Bool("filecustomrules_watch", false, "set up a watch on the target file and reload query rules when it changes")
+	// Commandline flag to specify rule path
+	fileRulePath        string
+	fileRuleShouldWatch bool
 )
 
+func registerFlags(fs *pflag.FlagSet) {
+	fs.StringVar(&fileRulePath, "filecustomrules", fileRulePath, "file based custom rule path")
+	fs.BoolVar(&fileRuleShouldWatch, "filecustomrules_watch", fileRuleShouldWatch, "set up a watch on the target file and reload query rules when it changes")
+}
+
+func init() {
+	servenv.OnParseFor("rulesctl", registerFlags)
```

Comment:
rulesctl uses cobra, and i will note that this flag did not appear in its `--help` output _before_ your change (it also doesn't show up afterwards): https://github.com/vitessio/vitess/blob/main/go/cmd/rulesctl/main.go#L14-L19